lib/viewmodels/about/about_viewmodel.dart

```diff
@@ -7,7 +7,7 @@ import 'package:mobile_app/viewmodels/base_viewmodel.dart';
 
 class AboutViewModel extends BaseModel {
   // ViewState Keys
-  final String FETCH_CONTRIBUTORS = 'fetch_contributors';
```

Comment:
I really liked the `ALL_CAPS_WITH_UNDERSCORES` naming convention but dart linter updated it to make lowerCamelCase. Though the dart rule also says, if we are already using ALL_CAPS_WITH_UNDERSCORES we can continue to use them but need to ignore this in linting. Refer: https://dart-lang.github.io/linter/lints/constant_identifier_names.html

This is really controversial decision and I will leave it to @Nitish145

Reply:
Can we request a review? 😅

Reply:
Yes, changing this format doesn't add any extra value to the codebase. So it is better to keep this rule disabled

Reply:
Sure, @Aadeesh11 you can disable `constant_identifier_names` and retain the ALL_CAPS_UNDERSCORE pattern wherever used.

Reply:
@manjotsidhu done, but had to create a new PR. Please review that.